exporter/metrics.go

```diff
@@ -427,13 +403,13 @@ func NewCounterMetric(cfg *configuration.MetricConfig, regex *oniguruma.Regex, d
 	}
 	if len(cfg.Labels) == 0 {
 		return &counterMetric{
-			metric:  newMetric(cfg, regex, deleteRegex),
+			observeMetric: newObserveMetric(cfg, regex, deleteRegex),
 			counter: prometheus.NewCounter(counterOpts),
 		}
 	} else {
 		return &counterVecMetric{
-			metricWithLabels: newMetricWithLabels(cfg, regex, deleteRegex),
-			counterVec:       prometheus.NewCounterVec(counterOpts, prometheusLabels(cfg.LabelTemplates)),
+			observeMetricWithLabels: newObserveMetricWithLabels(cfg, regex, deleteRegex),
+			counterVec:		 prometheus.NewCounterVec(counterOpts, prometheusLabels(cfg.LabelTemplates)),
```

Comment:
run `go fmt` on `metrics.go`

Reply:
Agreed, I've run `go fmt` in the coming commit.

I'm not a go developer whatsoever, so I did not know of the existence of `go fmt` until yesterday.